packages/beacon-node/test/utils/clock.ts

```diff
@@ -4,11 +4,14 @@ import {Slot, Epoch} from "@lodestar/types";
 import {IClock} from "../../src/util/clock.js";
 
 export class ClockStatic extends EventEmitter implements IClock {
+  genesisTime: number;
+
   constructor(
     readonly currentSlot: Slot,
-    public genesisTime = 0
+    genesisTime = 0
```

Comment:
why?

Reply:
To enforce consistent accessibility modifiers everywhere. 

We can either enforce explicitly specifying for every attribute or skip it for public. In our codebase most of the scenario were already following later case, so use `noPublic` option for that rule. 

https://biomejs.dev/linter/rules/use-consistent-member-accessibility/